ui/messages/en.json

```diff
@@ -91,23 +91,23 @@
       "offset": "Offset",
       "partition": "Partition",
       "size": "Size",
-      "timestamp": "Timestamp",
+      "timestamp": "Timestamp (local)",
+      "timestamp--utc": "Timestamp (UTC)",
```

Comment:
Just pointing out the two `-` here, in case it's a typo. Other keys in the file have underscores.

Reply:
Right, I did it like that it's because it's a variant, not a field. Translations are a mess and will have to redone anyway, I copy-pasted the old style from RHOSAK but it doesn't really work great with how this new i18n lib expects things to be.